types/suncalc/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strict": true,
```

Comment:
The Travis CI build will work just fine after you removed this option.

Reply:
Oh, I see.
I removed `strict` option.